apps/backend/src/router/auth.ts

```diff
@@ -60,11 +63,29 @@ router.get(
 router.get(
   '/google/callback',
   passport.authenticate('google', {
-    successRedirect: CLIENT_URL,
+    // successRedirect: CLIENT_URL,
```

Comment:
why we only comment this here and not on github or facebook too?

Reply:
Only tested for google, I'll update it after testing for github and facebook